packages/design-system/ui/src/components/calendar/calendar.tsx

```diff
@@ -61,7 +61,9 @@ const Calendar = ({
    */
   showOutsideDays = true,
   /**
-   * The locale to use for formatting dates. To change the locale pass a date-fns locale object.
+   * The locale to use for formatting dates. To change the locale pass a 
+   * [`date-fns` locale object](https://github.com/date-fns/date-fns/blob/main/docs/i18n.md). For 
+   * example, `{code: "en-US"}`
```

Comment:
@kasperkristensen Can you confirm this?

Reply:
It's like this:

```
import { es } from 'date-fns/locale'

<Calendar locale={es} {...} />
```

Reply:
Got it, then I'll add a dedicated example for it as that's probably clearer